lib/config/presets/internal/monorepos.ts

```diff
@@ -1,610 +1,35 @@
-  uppy: 'https://github.com/transloadit/uppy',
-  vaadinWebComponents: 'https://github.com/vaadin/web-components',
-  visx: 'https://github.com/airbnb/visx',
-  vitest: 'https://github.com/vitest-dev/vitest',
-  vstest: 'https://github.com/microsoft/vstest',
-  vue: ['https://github.com/vuejs/vue', 'https://github.com/vuejs/core'],
-  'vue-cli': 'https://github.com/vuejs/vue-cli',
-  vuepress: 'https://github.com/vuejs/vuepress',
-  weasel: 'https://github.com/JasperFx/weasel',
-  'web3-react': 'https://github.com/Uniswap/web3-react',
-  webdriverio: 'https://github.com/webdriverio/webdriverio',
-  wolverine: 'https://github.com/jasperfx/wolverine',
-  workbox: 'https://github.com/googlechrome/workbox',
-  xstate: 'https://github.com/statelyai/xstate',
-  xterm: 'https://github.com/xtermjs/xterm.js',
-  'xunit-dotnet': [
-    'https://github.com/xunit/assert.xunit',
-    'https://github.com/xunit/visualstudio.xunit',
-    'https://github.com/xunit/xunit',
-    'https://github.com/xunit/xunit.analyzers',
-  ],
-  yarn: 'https://github.com/yarnpkg/berry',
-  'zag-js': 'https://github.com/chakra-ui/zag',
-  'zxing-net': 'https://github.com/micjahn/ZXing.Net',
-};
-
-/* eslint sort-keys: ["error", "asc", {caseSensitive: false, natural: true}] */
-
-const orgGroups = {
-  hapijs: 'https://github.com/hapijs/',
-  lodash: 'https://github.com/lodash/',
-  ngrx: 'https://github.com/ngrx/',
-  nrwl: 'https://github.com/nrwl/',
-  octokit: 'https://github.com/octokit/',
-  'semantic-release': 'https://github.com/semantic-release/',
-  swc: 'https://github.com/swc-project/',
-};
-
-/* eslint sort-keys: ["error", "asc", {caseSensitive: false, natural: true}] */
-
-const patternGroups = {
-  angularmaterial: ['^@angular/material', '^@angular/cdk'],
-  'apache-camel': '^org.apache.camel:',
-  'aws-java-sdk': '^com.amazonaws:aws-java-sdk-',
-  'aws-java-sdk-v2': '^software.amazon.awssdk:',
-  babel6: '^babel6$',
-  clarity: ['^@cds/', '^@clr/'],
-  embroider: '^@embroider/',
-  forge: '^@forge/',
-  fullcalendar: '^@fullcalendar/',
-  hotchocolate: '^HotChocolate\\.',
-  'prometheus-simpleclient': '^io.prometheus:simpleclient',
-  // Can't specify the russh repo (https://github.com/warp-tech/russh) in repoGroups because parts
-  // of it (e.g. russh-config) are released separately.
-  russh: ['^russh$', '^russh-keys$'],
-  spfx: ['^@microsoft/sp-', '^@microsoft/eslint-.+-spfx$'],
-  spock: '^org\\.spockframework:spock-',
-  'syncfusion-dotnet': '^Syncfusion\\.',
-  'testing-library': '^@testing-library/',
-  wordpress: '^@wordpress/',
-};
-
-export const presets: Record<string, Preset> = {};
-
-for (const [name, value] of Object.entries(repoGroups)) {
-  presets[name] = {
-    description: `${name} monorepo`,
-    matchSourceUrls: toArray(value),
-  };
+async function getMonorepoPresets(): Promise<Record<string, Preset>> {
+  const presets: Record<string, Preset> = {};
+  const groups = JSON.parse(
+    await fs.readFile('lib/data/monorepo.json', 'utf8'),
+  );
```

Comment:
use static jsonc import, so this file is copied automatically. we also don't need async then